src/components/app/__tests__/App.test.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { mount, shallow, ReactWrapper } from 'enzyme';
+import { MemoryRouter } from 'react-router-dom';
+import Home from '../../home/Home';
+import App from '../App';
+
+function timeout(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
What is Promise and what does it do?

Reply:
[Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) is the underlying mechanism used by async / await for handling asynchronous code execution. A promise will eventually resolve (success) or reject (error) and return an appropriate value.